### DISCUSSION TITLE
Welcome to FPGA-Build Discussions!

### DISCUSSION THREAD

**Discussion by AugustinJose1221** [AugustinJose1221/FPGA-Build#2] [General]

**Comment by HACKER8HE** [comment]
Hello bro, after looking at your project I have to say that you are really a genius! But I've noticed that your project can only be simulated and not synthesized, so I would like to ask if you have been focusing on improving this project lately? I've also been working on SIFT-based binocular fusion recently, and we can talk about it

**Reply by AugustinJose1221** [reply]
Thanks for your interest in my work. This project was meant as a proof of concept for my undergraduate thesis. This project has been closed for the last two years and I never got the time to develop a synthesisable version. Now I find it hard to find  time to continue this work. I'd be happy to share the knowledge I acquired during course of this project.

**Comment by nirvana-ji** [comment]
Hello! After looking through your project, I have some questions to ask you. What does the "Template" folder in your files mean and how should it be used? Also, why is the Controller.v file in the "Template" folder different from the Controller.v file in the source files? I'm currently working on my undergraduate graduation project in this area and I'd really appreciate your help. Thank you very much.

**Comment by AugustinJose1221** [comment]
I have not done any refactoring to this project. Those files were initially developed and I used them as reference for further development. I'm pretty sure it can be refactored to a reasonable state

**Comment by 25172025** [comment]
Hello can I ask you one help how can the NoC FPGA code will be written for an image cnn operation
